### ISSUE TITLE
SCP Support

### ISSUE THREAD

**Issue by lockevod** [isontheline/pro.webssh.net#128]
**Describe the feature**
WebSSH is very useful and we've another included tools like sftp but I think it'll be more useful if we can have also scp enabled. thanks 

**Screenshots / Draws**
NA

**Comment by isontheline** [comment]
Hello @lockevod 👋

Thank you a lot for your issue!

Since SCP protocol has "less" possibility than SFTP protocol, your request is about a SCP command line?

Could you describe your use case please.

Have a nice day☀️

**Comment by lockevod** [comment]
Yes, i know sftp is more powerful than scp but you've to setup a sftp service (and sometimes you aren't server owner). 
Command line is useful but i was thinking something like winscp.

Regards

**Comment by isontheline** [comment]
OK I agree

Can you confirm me that you can also run SSH commands on your server?

Because SCP doesn't support listing, I need to implement listing as "ls" command like WinSCP does :
https://winscp.net/eng/docs/requirements#scp

**Comment by isontheline** [comment]
@lockevod Any confirmation about launching SSH commands in order to implement SCP missing features?

**Comment by isontheline** [comment]
https://github.com/isontheline/pro.webssh.net/discussions/255#discussion-3316702

**Comment by klue-less** [comment]
Adding SCP is a very good idea.

**Comment by isontheline** [comment]
@klue-less Are you OK that WebSSH will send SSH commands to your server in order to operate SCP listing directories?

**Comment by primovist** [comment]
Synology only supports SCP not SFTP.

**Comment by isontheline** [comment]
Hello @primovist 👋

I will implement SCP protocol in a near release

**Comment by diziiiii** [comment]
Hope WebSSH supports SCP protocol soon.

**Comment by isontheline** [comment]
Hey folks 👋

This feature can now be tested 🎉
👉 https://testflight.apple.com/join/QSrBK59z

Feedback is welcome 🙏

<details>
<summary>Want to help me to support WebSSH further?</summary>

WebSSH isn't a subscription based app (you buy it for [lifetime](https://webssh.net/documentation/pricing/)) but if you like my work and want to support me further, you can become a [sponsor](https://github.com/sponsors/isontheline) ⭐ or you can just buy me a tiny [coffee](https://www.buymeacoffee.com/isontheline) ☕️

And, if you enjoyed using WebSSH, please leave a review on the [App Store](https://apps.apple.com/app/apple-store/id497714887?action=write-review)! It really helps me out. Thanks!

</details>
